.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,61 @@
+name: Integration tests
+on:
+  pull_request:
+  push:
+    branches: [main]
```

Comment:
This will run on every PR, as well as merges to main.

Reply:
Love it! ❤️